build.gradle

```diff
@@ -108,7 +112,7 @@ ext.externalDependency = [
   "hadoopYarnClient": "org.apache.hadoop:hadoop-yarn-client:"+hadoopVersion,
   "hadoopYarnMiniCluster": "org.apache.hadoop:hadoop-minicluster:"+hadoopVersion,
   "hadoopAnnotations": "org.apache.hadoop:hadoop-annotations:"+hadoopVersion,
-  "hadoopAws": "org.apache.hadoop:hadoop-aws:2.6.0",
+  "hadoopAws": "org.apache.hadoop:hadoop-aws:"+hadoopAwsVersion,
```

Comment:
I know it isn't but there is the possibility to use other versions. For instance, we are running in Qubole and they have their own version.